python/mrtrix3/commands/5ttgen/__init__.py

```diff
@@ -14,4 +14,4 @@
 # For more details, see http://www.mrtrix.org/.
 
 # pylint: disable=unused-variable
-ALGORITHMS = ['freesurfer', 'fsl', 'gif', 'hsvs']
+ALGORITHMS = ['freesurfer', 'fsl', 'gif', 'hsvs', 'msmt']
```

Comment:
@arkiev Code would not have been possible to execute via `5ttgen` until it was added here.